proposal.rst

```diff
@@ -272,34 +287,138 @@ It would still produce better estimates as we reduced the number of documents we
 Project Timeline
 ----------------
 
-* Three Weeks of Bonding:
+The main goal of this project is to improve the estimation of the total number of results buy making use of the
+the new feature that gives the first and last docids for each term.
+
+The task is to improve the Xapian::MSet::get_matches_estimated() method for 17 operators, but OP_AND and OP_OR are almost done, so that leave us with 15 operators.
```

Comment:
As I said on chat, I think you want to consider the `PostList` subclasses rather than the operators.  In some cases they are pretty much equivalent, but not in all cases.